workflow/signal/signal.go

```diff
@@ -16,6 +16,9 @@ import (
 
 func SignalContainer(restConfig *rest.Config, pod *corev1.Pod, container string, s syscall.Signal) error {
 	command := []string{"/bin/sh", "-c", "kill -%d 1"}
+	if container == common.WaitContainerName {
```

Comment:
nice - I was about to say you should annotate this kill command - but this is backwards compatible